precise_driver/package.xml

```diff
@@ -12,13 +12,18 @@
   <buildtool_depend>catkin</buildtool_depend>
 
   <depend>angles</depend>
+  <depend>actionlib_msgs</depend>
   <depend>cob_srvs</depend>
+  <depend>controller_manager_msgs</depend>
   <depend>geometry_msgs</depend>
+  <depend>message_generation</depend>
   <depend>realtime_tools</depend>
   <depend>ros_control_boilerplate</depend>
+  <depend>std_msgs</depend>
   <depend>std_srvs</depend>
 
   <exec_depend>controller_manager</exec_depend>
+  <exec_depend>message_runtime</exec_depend>
   <exec_depend>precise_description</exec_depend>
```

Comment:
`joint_state_controller` and `position_controllers` should be listed as `exec_depend` - from the the `controller.yaml`